### PR TITLE
PC-65 Close dropdown menu

### PR DESCRIPTION
**Pull request by KenAdams777**
Implemented closing dropdown by clicking menu links and outside menu.
Also added a dialog window to confirm logging out.
You are welcome to leave your comments and suggestions.

### CODE REVIEW

File: src/components/Menu/Menu.js [hunk 1]
```diff
@@ -1,42 +1,65 @@
...
-					<li>
-						<Link to="/autors">Authors</Link>
-					</li>
-					<li>
-						<Link to="/bookmarks">Bookmarks</Link>
-					</li>
-					<li>
-						<Link to="/my-profile">My&nbsp;Profile</Link>
-					</li>
+					<Link to="/autors">
```
Comment (MuscularSloth) [inline src/components/Menu/Menu.js:38]: I thik we should continue use the `ul>li>a` structure cause this is semanticaly right way. So the `Link` should be inside a `li`, shouldn't it?

If you wanna stretch the `a` to the entire `li`, you can change the `a` to a block element and set "height: 100%" if necessary.

File: src/components/Modals/Login/Login.js [hunk 2]
```diff
@@ -88,11 +88,11 @@ function Login() {
 
 	useEffect(() => {
 		dispatch(getBookmarksAsyncAction());
-	}, [isLoginSuccess])
+	}, [isLoginSuccess]);
```
Comment (MuscularSloth) [inline src/components/Modals/Login/Login.js:91]: I know it is not part of your task, but I noticed, that we have several different `useEffect` hooks for the same trigger `isLoginSuccess`. (row 91 and row 100) 
I'm not sure if it's a good practice to do. Maybe we should combine them?

**Review by MuscularSloth — COMMENTED** [review]
Good job! Please take a look at the comments below =)

File: src/components/BurgerMenu/BurgerMenu.js [hunk 3]
```diff
@@ -10,12 +10,17 @@ function BurgerMenu({ handleClick, isMenuOpen }) {
 	const menuToggle = isMenuOpen ? menuStates.OPEN : menuStates.CLOSE;
```
Comment (bejalane) [inline src/components/BurgerMenu/BurgerMenu.js:10]: can't understanf why this is not boolean?

**Review by bejalane — APPROVED** [review]

**Review by MuscularSloth — APPROVED** [review]

**Pull request merged**